tests/integration/helpers/s3_tools.py

```diff
@@ -54,6 +54,42 @@ def upload_file(self, local_path, remote_blob_path, bucket=None):
             file_path=local_path,
         )
 
+class S3Downloader:
```

Comment:
It would be nice to add such downloader in all other tests, but I'll do it by myself